runtimebp/internal/maxprocs/maxprocs.go

```diff
@@ -0,0 +1,111 @@
+// Package maxprocs optimizes GOMAXPROCS for Infrared environments.
+// It attempts to balance application performance with cluster-level resource utilization.
+// Applications may tune GOMAXPROCS as necessary (see Set).
+// The defaults in this package are subject to change.
+package maxprocs
+
```

Comment:
maybe include a note that the "canonical" version of this is not here?

```suggestion

// Note: this is manually copied from internal baseplate/v2/internal/maxprocs/maxprocs.go

```